wheel/src/api.rs

```diff
@@ -78,10 +78,11 @@ use chia_bls::{
 #[pyfunction]
 pub fn compute_merkle_set_root<'p>(
     py: Python<'p>,
-    values: Vec<&'p PyBytes>,
+    values: Vec<Bound<'p, PyBytes>>,
```

Comment:
The loss of the `&` makes me squirmy.  Adding it results in an error.

```rust
error[E0277]: the trait bound `Vec<&pyo3::Bound<'_, PyBytes>>: PyFunctionArgument<'_, '_>` is not satisfied
   --> wheel/src/api.rs:81:13
    |
81  |     values: Vec<&Bound<'p, PyBytes>>,
    |             ^^^ the trait `FromPyObject<'_>` is not implemented for `Vec<&pyo3::Bound<'_, PyBytes>>`, which is required by `Vec<&pyo3::Bound<'_, PyBytes>>: PyFunctionArgument<'_, '_>`
    |
    = help: the trait `FromPyObject<'py>` is implemented for `Vec<T>`
    = note: required for `Vec<&pyo3::Bound<'_, PyBytes>>` to implement `FromPyObjectBound<'_, '_>`
    = note: required for `Vec<&pyo3::Bound<'_, PyBytes>>` to implement `PyFunctionArgument<'_, '_>`
note: required by a bound in `extract_argument`
   --> /home/altendky/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pyo3-0.22.5/src/impl_/extract_argument.rs:119:8
    |
113 | pub fn extract_argument<'a, 'py, T>(
    |        ---------------- required by a bound in this function
...
119 |     T: PyFunctionArgument<'a, 'py>,
    |        ^^^^^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `extract_argument`
```

Reply:
The size of `Bound` 8 bytes (ie it's a pointer), so I'm not too worried about it